### DISCUSSION TITLE
batch export direct messages.

### DISCUSSION THREAD

**Discussion by AhmedTawfeeq** [Tyrrrz/DiscordChatExporter#568] [General]
The program is great overall but one thing interests me still. 

Is there some way to batch save direct messages? Like so i can export DM's with 10 Persons in one go? Because it would be a pain for some one who wrote with 100 different people to export them one by one.

**Comment by Tyrrrz** [comment]
Yes, you can multiselect with shift in GUI or pass multiple channel IDs in CLI

**Comment by halo4223** [comment]
Export Direct Messages would be super fine.
I would appreciate that very much
